openshift-kubecost-install.md

```diff
@@ -0,0 +1,513 @@
+Install Kubecost on Redhat Openshift cluster
```

Comment:
The "O" and "S" in OpenShift should be capitalized. This includes any diagrams.

Reply:
I would do a find and replace over the entire doc.

Reply:
Thanks Sean ! It is updated in the latest commit.